content/virtualization.md

```diff
@@ -0,0 +1,244 @@
+
+#### VD Agent
+
+- Allows the VM screen to resize automatically to match the GNOME boxes window
+- Provides ability to copy/paste between the Host and VM
+- Support for Dragging and Dropping files from the Host to the VM.
+
+#### WebDAVd
+
+- Share folders from the Host to the VM.
+
+> The driver names may vary by distribution, as shown in the below Arch example: phodav is used instead of the spice-webdavd package name. We recommend referencing your specific distributions documentation for equivalent or alternative driver names
+
+#### Install the drivers on popular Linux distributions
+
+- Pop
+
+```bash
+sudo apt update
+sudo apt install spice-vdagent spice-webdavd
+```
+
+- Ubuntu
+
+```bash
+sudo apt update
+sudo apt install spice-webdavd
+```
+
+- Fedora
+
+```bash
+sudo dnf install spice-webdavd
+```
+
+- Arch
+
+```bash
+sudo pacman -Sy spice-vdagent phodav
+```
+
+### Windows Guests
+
+On Windows, there are also 2 drivers that will need installed for all the features of GNOME Boxes to be supported by Windows. These drivers are available on the [Spice Project's](https://www.spice-space.org/download.html#windows-binaries) download page. There are a number of downloads listed in this section, but the 2 drivers we need are the [Windows guest tools](https://gitlab.freedesktop.org/spice/win32/spice-nsis) and the [Spice WebDAVd](https://git.gnome.org/browse/phodav/tree/spice) drivers.
+
+### Windows Driver Downloads
+
+- [Windows Guest Tools](https://www.spice-space.org/download/windows/spice-guest-tools/spice-guest-tools-latest.exe)
+- [Spice WebDAVd](https://www.spice-space.org/download/windows/spice-webdavd/spice-webdavd-x64-latest.msi)
+
+> If you are unable to download either of the above files, right click the link above and select "Save Link As..." and save the file to your downloads folder.
+
+## Virtualization Features
+
+Now that the VM is set up, and drivers are installed, the advanced features of GNOME Boxes are now available.
+
+### VM Properties Menu
+
+Most of these features can be found in the VM's properties menu. To reach the VM's properties menu:
+
+1. Start your VM by clicking on the VM in the GNOME Boxes list.
+2. Select the <kbd>&vellip;</kbd> button at the top right.
+3. Select "Properties" to open the properties menu.
+
+### Copy and Paste
+
+Once your VM has the `spice-vdagent` or an equivalent package installed on your VM, reboot the VM once to load this driver. At this point, you will now be able to copy and paste between your VM and the Host just like copy and pasting between applications.
+
+### Share a USB Device
+
+![Devices and Shares](/images/gnome-boxes/boxes-vmprefs-usb.png)
+
+One of the features of GNOME Boxes, is the ability to transfer a USB device connected to your Host to the VM. To share a USB device
```

Comment:
missing colon after "To share a USB device"

Reply:
There should not be a comma after "GNOME Boxes" in this sentence, either.